mod_reforged/hooks/skills/racial/champion_racial.nut

```diff
@@ -0,0 +1,80 @@
+::mods_hookExactClass("skills/racial/champion_racial", function(o) {
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/damage_dealt.png",
```

Comment:
Vanilla is rather inconsistent with this icon. For almost all cases it uses `regular_damage.png` for damage buffs. But in two cases, it uses `damage_dealt.png`. I personally prefer the use of `regular_damage.png` as the `damage_dealt.png` looks somewhat similar to `melee_skill.png`. But you can decide what you prefer here.